lib/organization.js

```diff
@@ -123,19 +133,18 @@ export class Organization {
       initialSchemesParams,
       initialSchemesTokenAddresses,
       initialSchemesFees,
-      initialSchemesPermissions,
+      initialSchemesPermissions
     );
 
-
     // transfer what we need for fees to register the organization at the given schemes
     // TODO: check if we have the funds, if not, throw an exception
     // fee = await org.schemeRegistrar.fee())
     // we must do this after setSchemes, because that one approves the transactions
     // (but that logic shoudl change)
-    for (let i=0; i < initialSchemesAddresses.length; i = i + 1) {
+    for (let i = 0; i < initialSchemesAddresses.length; i = i + 1) {
       const scheme = await SchemeRegistrar.at(initialSchemesAddresses[i]);
       const token = await DAOToken.at(initialSchemesTokenAddresses[i]);
-      const fee  = initialSchemesFees[i];
+      const fee = initialSchemesFees[i];
       await token.transfer(org.avatar.address, fee);
       await scheme.registerOrganization(org.avatar.address);
```

Comment:
This is not needed any more as the registerOrganization is done in setSchemes for the case it is  universalScheme.
Take a look at daostack https://github.com/daostack/daostack/blob/master/contracts/universalSchemes/GenesisScheme.sol#L88